src/main/presenter/mcpPresenter/inMemoryServers/appleServer.ts

```
@@ -1745,9 +1870,12 @@ export class AppleServer {
             content: [
               {
                 type: 'text' as const,
-                text: `Found ${listReminders.length} reminders in list ${parsedArgs.listId}:\n\n${listReminders.map(reminder => 
-                  `• ${reminder.name} - ${reminder.completed ? 'Completed' : 'Pending'}`
-                ).join('\n')}`
+                text: `Found ${listReminders.length} reminders in list ${parsedArgs.listId}:\n\n${listReminders
+                  .map(
+                    (reminder) =>
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)